src/main/java/com/passpnu/passwordmanager/generator/StringPasswordGenerator.java

```diff
@@ -0,0 +1,35 @@
+package com.passpnu.passwordmanager.generator;
+
+import org.passay.CharacterRule;
+import org.passay.EnglishCharacterData;
+import org.passay.PasswordGenerator;
+import org.springframework.stereotype.Component;
+
+import java.security.SecureRandom;
+
+@Component
+public class StringPasswordGenerator {
+    private StringPasswordGenerator(){}
+
+    public static String generate(){
```

Comment:
Some special characters would be nice since enterprise password complexity standards require those

Reply:
fixed